pkg/admin/installer/directpv.min.io_directpvdrives.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.15.0
+    controller-gen.kubebuilder.io/version: v0.14.0
```

Comment:
please update your controller-gen version and run `build.sh` again

Reply:
Okay, i did that already.